docs/create/azure/portal/index-azure-portal.mdx

```diff
@@ -0,0 +1,52 @@
+---
+id: index-azure-portal
+title: Create Redis database on Azure Cache
+sidebar_label: Redis on Azure Cache
+slug: /create/azure/portal
+---
+
+Azure Cache for Redis is a native fully-managed service on Microsoft Azure. Azure Cache for Redis offers both the Redis open-source (OSS Redis) and a commercial product from Redis Labs (Redis Enterprise) as a managed service. It provides secure and dedicated Redis server instances and full Redis API compatibility. The service is operated by Microsoft, hosted on Azure, and accessible to any application within or outside of Azure.
+
+
+
+### Step 1. Getting Started 
+
+
+Launch  [Azure Cache for Redis Enterprise & Flash](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/garantiadata.redis_enterprise_1sp_public_preview?ocid=redisga_redislabs_cloudpartner_cta1) 
+
+
+![RedisLabs Azure Page](azure1.png) 
+
+
+### Step 2: Setup & Subscribe
+
+![RedisLabs Azure Page](azure2.png)
+
+
+### Step 3: Configuring New Redis Cache
+
+![RedisLabs Azure Page](azure3.png)
+
+
+### Step 4: Finalising the setup
+
+![RedisLabs Azure Page](azure4.png)
+
+
+### Step 5: Connecting to Redis database
+
+```
+sudo redis-cli -h redislabs.redis.cache.windows.net -p 6379
+redislabs.redis.cache.windows.net:6379>
+```
+
+### Resources
+
+<div class="text--center">
+<iframe width="760" height="415" src="https://www.youtube.com/embed/eThj-jwViZw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+
+### Next Step
+
+- [Best Practices for Azure Cache for Redis](https://docs.microsoft.com/en-in/azure/azure-cache-for-redis/cache-best-practices)
+- [Quickstart: Use Azure Cache for Redis in .NET Framework](https://docs.microsoft.com/en-us/azure/azure-cache-for-redis/cache-dotnet-how-to-use-azure-redis-cache)
```

Comment:
Add a link to https://developer.redislabs.com/develop/dotnet/ as well in here.

Reply:
Added.

Reply:
Added.